config/src/main/java/com/alibaba/nacos/config/server/service/kubernetes/ConfigMapSyncConfig.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * Copyright 1999-2023 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.alibaba.nacos.config.server.service.kubernetes;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * ConfigMap synchronization configuration.
+ *
+ * @author wangyixing
+ */
+@Component
+public class ConfigMapSyncConfig {
+    
+    @Value("${nacos.k8s.configMap.enabled:false}")
+    private boolean enabled;
+    
+    @Value("${nacos.k8s.configMap.responsible:false}")
```

Comment:
unused configuration.
What is the meaning of the configuration item?

Reply:
`enabled` is the on-off switch of configMap synchronization job.
`responsible` is designed to decide which node is responsible for data persistence job.But we did not have time to develop this part, so I leave the config for future TODO.